### DISCUSSION TITLE
Subgroup Ideogram Marker Colors and Edge Colors Not Matching

### DISCUSSION THREAD

**Discussion by gennachiaro** [NMGRL/pychron#2167] [Q&A] [answered]
Hi,

When I plot up my data as a subgroup ideogram, the marker colors do not match the edge colors.  In Groups, I select "bind colors" and have tried experimenting with making the line widths zero, white (and not binding the colors), and switching to other pre-made graph templates.  The problem persists throughout all scenarios.

<img width="1024" alt="Screen Shot 2022-04-19 at 11 40 08 PM" src="https://user-images.githubusercontent.com/56849646/164154525-9a92a5b8-6b7d-4824-ad99-48cd8a5354a7.png">

Also, when I omit datapoints, there is just an open-face square behind the original symbol, so it is not clear whether the datapoint is omitted or not (zoom in on leftmost datapoint on plot below).  

<img width="989" alt="Screen Shot 2022-04-19 at 11 24 48 PM" src="https://user-images.githubusercontent.com/56849646/164154555-fa38ccfe-f74b-4d49-a1c2-fa24e66812b5.png">

How should I go about fixing this?

Thank you!

**Comment by jirhiker** [comment, marked as answer]
Try turning off Show Subgroup Indicators in Ideogram/Appearance.

For now this would have to be resolved in post via Illustrator or the like.

**Reply by jirhiker** [reply]
Follow up question, do you really want to be using a subgrouped ideogram?  subgrouping is typically only useful when you want to plot the plateau/isochron/integrated ages for step heat experiments as individual analyses on an ideogram.

**Reply by jirhiker** [reply]
When plotting plateau dates (aka subgroup dates) with fusion dates the subgroups are identified with different symbols. 
Try turning off ```Show Subgroup Indicators``` in Ideogram/Appearance.

**Reply by gennachiaro** [reply]
Ok, will fix in illustrator!  And yes, all my crystals were step-heated so I have been plotting these as valid integrated ages.  

Turning off the "Show Subgroup Indicators" actually solved every problem!!  The only thing is that when one analysis is omitted, it shows as a square regardless of the sample (but not a big problem at all).  
![ORA-2A_5B](https://user-images.githubusercontent.com/56849646/164266629-e4e24e41-dbfd-4fc9-8bd8-410e1c2b00ca.png)


Thank you!

**Reply by jirhiker** [reply]
Excellent.  
" The only thing is that when one analysis is omitted, it shows as a square regardless of the sample"  This will be resolved in a future release. Thanks
